netbout-client/src/test/java/com/netbout/client/RtAliasITCase.java

```diff
@@ -58,8 +58,17 @@ public void updateAndRetrieveEmail() throws Exception {
         final MkBase base = new MkBase();
         final User user = NbRule.get();
         final Alias alias = user.aliases().iterate().iterator().next();
-        final String email = base.randomAlias().email();
-        alias.email(email);
-        MatcherAssert.assertThat(alias.email(), Matchers.is(email));
+        final String verified;
+        if (alias.email().contains("!")) {
```

Comment:
@CarlosGines I suggest to use `alias.email().indexOf()`, extract it to a variable to reuse it in L63, and test if it is -1 in the condition.